scripts/apps/search-providers/directive.ts

```diff
@@ -11,6 +11,12 @@ export default function SearchProviderConfigDirective(searchProviderService, not
             $scope.newDestination = null;
             $scope.gettext = gettext;
 
+            $scope.availableListViews = {
+                '': {label: gettext('None')},
+                list: {label: gettext('List View'), icon: 'th-list'},
+                photogrid: {label: gettext('Photo Grid View'), icon: 'th'},
```

Comment:
I'd call it simply "grid" since it's not limited to working with photos.

Reply:
That's how the ticket required the button to be named. I guess they are used to it and also it's very verbose because even if they are not images you do see images in the grid

Reply:
I was referring to the object key name. Labels visible to users are easy to change so I'm not concerned about them.